services/core/shared/core/compat/sdk_v2/index.js

```diff
@@ -135,12 +135,11 @@ const getAddressByAny = async param => {
 		const parsedAddress = parseAddress(param);
 		if (validateAddress(parsedAddress)
 			&& await confirmAddress(parsedAddress)) return parsedAddress;
-		if (validatePublicKey(param)) return getAddressByPublicKey(param);
-		return getAddressByUsername(param);
 	}
 
 	const [prefix, body] = separateParam(param);
-	return paramNames[prefix](body);
+	if (prefix || body) return paramNames[prefix](body);
```

Comment:
Checking for `body` within the condition, doesn't make sense if `prefix` is returned to be `undefined`.

```suggestion
	if (prefix && body) return paramNames[prefix](body);
```
or:
```suggestion
	if (prefix) return paramNames[prefix](body);
```

Reply:
Done!